Makefile

```diff
@@ -37,16 +37,42 @@ clean:
 	$(MAKE) -C man clean
 	$(MAKE) -C src clean
 
+INSTALL=install
+INSTALL_PROGRAM=$(INSTALL)
+INSTALL_DATA=$(INSTALL) -m 644
+
+prefix = /usr/local
+exec_prefix = $(prefix)
+bindir = $(exec_prefix)/bin
+datarootdir = $(prefix)/share
+mandir = $(datarootdir)/man
+man1dir = $(mandir)/man1
+manext = .1
+
 .PHONY: install
-install:
-	@printf "\n\n=========================================\n\
-To install, copy the files src/unison, src/unison-gui (optional),\n\
-src/unison-fsmonitor (optional) and src/fsmonitor.py (optional,\n\
-if unison-fsmonitor does not exist) to a freely chosen location.\n\n\
-Manual page is at man/unison.1 and user manual is at\n\
-doc/unison-manual.pdf, doc/unison-manual.html and doc/unison-manual.txt\n\
-=========================================\n\n\n"
-	@exit 1
+install: all
+	$(INSTALL) -d "$(DESTDIR)$(bindir)"
+ifneq ($(wildcard src/unison),)
+	$(INSTALL_PROGRAM) src/unison "$(DESTDIR)$(bindir)/unison"
```

Comment:
bindir is not mkdir -d'd.  I know that sounds funny, but I think it's normal to install a program into its own prefix for testing.

Reply:
`$(INSTALL) -d` on line 54 is supposed to do that here. (Also tested precisely as you describe.)